packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testLabel.st

```diff
@@ -0,0 +1,13 @@
+tests
+testLabel
+	
+	| labelActual |
+	
+	"TODO discuss if this should be kept"
```

Comment:
The regex test covers everything that the substring tests cover and additionally also checks that the label has the correct overall format.
I think we should remove the substring tests in favor of the regex test.
The `endsWith` test is a gread addition to this test method.

Reply:
It is true that the regex test is probably covering more than this.
Our reasons for adopting this approach were:
1. Not all Squeak-Versions are capable of interpreting the regex, since this is not in the core but part of the `vivide`-package. This lead to be build failing on 5.0. We then had the choice to add the necessary methods to our project or use this version.
2. The regex is hard to understand and we figured a test should be easy to understand.

Reply:
I see your point, thanks for the explanation. Still I would suggest somehow testing the format of the label. I don't have time to think about this right now, please let me know if you can take care of it or if I should.